package.json

```diff
@@ -205,6 +204,8 @@
     "**/**/ini": "^1.3.6",
     "**/**/elliptic": "^6.5.4",
     "**/**/y18n": "^3.2.2",
-    "**/**/jose": "1.28.1"
+    "**/**/jose": "1.28.1",
+    "**/**/**/ssri": "^8.0.1",
```

Comment:
@leepavlouhmcts - same question as on the other PR, regarding the number of `**/`'s

Reply:
replaced **/**./**/** to **/**/